.travis.yml

```diff
@@ -19,7 +19,7 @@ env:
   matrix:
     - TEST_SUITE=test
     - TEST_SUITE=cucumber
-
+    - TEST_SUITE=default-site
```

Comment:
To add it only once, choose your fave Ruby version and add it to the `matrix.include` section above this one (see `fmt`):

```yaml
- rvm: 2.3.0
  env: TEST_SUITE=test-jekyll-new
```

Or whatever 😄